src/styles/utils/variables.scss

```diff
@@ -0,0 +1,10 @@
+$hover-text-color: #00acdc;
+$main-font-family: "Roboto", sans-serif;
+$main-text-color: #060b35;
+$second-cards-color: #616070;
+$main-border-color: #f3f3f3;
+$white-c: #fff;
```

Comment:
It is better to try not to name variables according to their content, but to name them according to their meaning. This will improve the maintainability of your code.